packages/frontend/src/containers/AboutUsModalContainer.tsx

```diff
@@ -0,0 +1,17 @@
+import { toggleAboutModal } from "src/api/store";
+import { useAppDispatch, useAppSelector } from "src/api/store/hooks";
+import { AboutUsModal } from "src/components/AboutUsModal";
+
+export const AboutUsModalContainer = () => {
+    const dispatch = useAppDispatch();
+    const showModal = useAppSelector((state) => state.ui.showAboutModal);
+    const toggleModal = () => dispatch(toggleAboutModal());
+
+    const onClose = () => {
+        if (showModal) {
+            toggleModal();
```

Comment:
Any reason why we're persisting this to the state and not using something more transient?

Reply:
This doesn't need persistence, but to keep the code well organised it does make sense to keep the state in redux. It just happens that the whole redux store is persisted. 
Something similar applies to UI state variables like `showWidgetLib`